bundle/edu.gemini.lchquery.servlet/src/main/scala/edu/gemini/lchquery/servlet/LchQueryFunctor.scala

```diff
@@ -81,7 +81,7 @@ class LchQueryFunctor(queryType: LchQueryFunctor.QueryType,
             n <- target.getNonSiderealTarget
             h <- n.horizonsDesignation
           } {
-            setHorizonsObjectId(h.queryString)
```

Comment:
The change seems trivial enough but I don't know how this is used.  Presumably something will parse the formatted horizons designation and use it to query horizons?  Is that done?

Reply:
Yes, we have the `HorizonsDesignation` class, which will parse the output of `show` using `read` into a query string that can be used to uniquely identify a Horizons object.

Reply:
(And that query string will be used in a query URL to Horizons to get the necessary data.)